### PR TITLE
updates to use documentation address ranges

### PR DESCRIPTION
**Pull request by sulrich**
this is a reasonably comprehensive first pass at removing references to non-documentation address ranges from the repo.

addresses: 
- #97

### CODE REVIEW

**Comment by noredistribution** [comment]
found a few more in `contributing/content-style-guide.md` and `docs/examples/gnmi-clients/gnmic/outputs/interface_state.json`

should we also change it in `docs/telemetry/adapters/ocprometheus/index.md` and `docs/examples/netconf/ncclient.md` and lots of `.py` files in `src/ncclient` ?

**Comment by sulrich** [comment]
yeah, there's more to be done, but i figured if i have an open PR for it, i'll be motivated to grind through it over beer. 😉

File: docs/examples/gnmi-clients/arista-gnmi/index.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ categories:
...
 ```shell
-gnmi -addr 10.83.13.139:6030 -username admin -password arista get /
+gnmi -addr 192.0.2.139:6030 -username admin -password arista get /
 ```
 
 #### Get the BGP configuration in the default VRF
 
 ```shell
-gnmi -addr 172.28.160.219:6030 -username admin -password arista \
+gnmi -addr 198.51.100.219:6030 -username admin -password arista \
```
Comment (fredhsu) [inline docs/examples/gnmi-clients/arista-gnmi/index.md:21]: should be 192.0.2.100?

File: docs/examples/gnmi-clients/arista-gnmi/index.md [hunk 2]
```diff
@@ -799,59 +799,59 @@ gnmi -addr 10.83.13.139:6030 -username admin get origin=eos_native '/Sysdb/conne
 ##### Configure neighbor address and peer AS
 
 ```shell
-gnmi -addr 10.83.13.139:6030 -username cvpadmin -password arastra \
+gnmi -addr 192.0.2.139:6030 -username cvpadmin -password arista \
   update '/network-instances/network-instance[name=default]/protocols/protocol[name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.10.100.43]' \
```
Comment (fredhsu) [inline docs/examples/gnmi-clients/arista-gnmi/index.md:803]: are we concerned with bgp peer addresses?

Comment (sulrich) [reply docs/examples/gnmi-clients/arista-gnmi/index.md]: yes, this is in sample CLI.  i’ve updated this. (hopefully)

**Review by fredhsu — APPROVED** [review]

**Pull request merged**